### PR TITLE
limited input field to number

### PR DESCRIPTION
**Pull request by mAkeddar**
Known limitations:
Users can still enter the letter e as it considers numbers like 1e-5

### CODE REVIEW

File: src/Pages/Sender/EnterPrice.tsx [hunk 1]
```diff
@@ -60,6 +60,8 @@ const EnterPrice: React.FC = () => {
               value={objecInfo.price !== 0 ? objecInfo.price : undefined}
               onChange={handleInputChange}
               placeholder={t('requestCost.pricePlaceholder') || ''}
+              suffix="CHF"
```
Comment (mischakolbe) [inline src/Pages/Sender/EnterPrice.tsx:63]: The suffix is cool to have! We could even get rid of the "(in CHF)" in the title...

File: src/Pages/Sender/EnterPrice.tsx [hunk 2]
```diff
@@ -60,6 +60,8 @@ const EnterPrice: React.FC = () => {
...
+              type="number"
```
Comment (mischakolbe) [inline src/Pages/Sender/EnterPrice.tsx:64]: This doesn't fully do what I think it should: I can still enter letters (when I start with letters and then numbers - doesn't work the other way around: numbers and then letters wipes the field), but then it simply goes away entirely when I go to the next page:

image
[NEXT PAGE]
image

In fact; it doesn't allow me to enter a dot or comma - that wipes the number altogether as well.

**Review by mischakolbe — CHANGES_REQUESTED** [review]
Doesn't seem to truly fix the issue atm.

File: src/Pages/Sender/EnterPrice.tsx (hunk 2 above)
Comment (mAkeddar) [reply src/Pages/Sender/EnterPrice.tsx:64]: This one is strange, it's working on Chrome.
Just tried on Firefox and I have the same behavior as you
Are you using Firefox/safari?

Comment (mAkeddar) [reply src/Pages/Sender/EnterPrice.tsx]: The new push should fix it ;)

**Pull request merged**